app/src/main/java/com/example/newsapp/data/dao/UserDao.kt

```diff
@@ -9,7 +9,7 @@ import com.example.newsapp.data.entity.User
 interface UserDao {
 
     @Query("select * from userInfo where uid=1")
-    fun getUser(): User
+    fun getUser(): List<User>
```

Comment:
Since we are supposed to have one user in the DB, I think we can keep a single object as a return value instead of a list.